generation/new_client/new-client.py

```diff
@@ -250,7 +250,7 @@ def generate(
     )
 
     # In monorepo, .OwlBot.yaml needs to be in the directory of the module.
-    owlbot_yaml_location_from_module = ".OwlBot.yaml"
+    owlbot_yaml_location_from_module = ".OwlBot-hermetic.yaml"
```

Comment:
I purposely didn't change this in my PR because `new-client.py` is the old process and should be removed soon.

Reply:
Sure, I'll restore this change.

Reply:
Done.